tf/tfprocess.py

```diff
@@ -220,6 +276,14 @@ def __init__(self, cfg):
             raise ValueError(
                 "Unknown default activation type: {}".format(default_activation))
 
+        if self.encoder_layers > 0:
+            self.net.set_networkformat(
+                pb.NetworkFormat.NETWORK_ATTENTIONBODY_WITH_HEADFORMAT)
+
+        self.net.set_smolgen_activation(self.net.activation(self.smolgen_activation))
```

Comment:
These two sets up the minimum version of the network - you should guard them by the network format being attention body with head format.

Reply:
not sure what is wanted here, so i put the two activations into the above if statement